source/Company.py

```diff
@@ -1,53 +1,184 @@
+# Company Class exercise
+
+def contains_number(value):
+    '''
+    A function which checks if a value includes digits
+    '''
+    if True in [char.isdigit() for char in value]:
+        return True
+    return False
+
+
+def check_string(str: str):
+    '''
+    A function which checks if the conditions of
+    company name are met
+    '''
+    if contains_number(str) or len(str) < 2:
+        return False
+    elif str[0].isupper() and len(str) >= 2:
+        return True
```

Comment:
What happens when both if statements result in a False? what will be returned?
Always make sure your functions has a return for all code paths.
example: for "ofek", None will be returned.

Reply:
fixed